src/applications/check-in/pages/Confirmation.jsx

```diff
@@ -8,12 +9,9 @@ import Footer from '../components/Footer';
 import AppointmentLocation from '../components/AppointmentLocation';
 
 const Confirmation = () => {
-  useEffect(() => {
-    focusElement('h1');
-  }, []);
   return (
     <div className="vads-l-grid-container vads-u-padding-y--5">
-      <va-alert status="success">
+      <VaAlert status="success" onVaOnLoad={() => focusElement('h1')}>
```

Comment:
For posterity: the `onVaOnLoad` event handler won't work until we add it to the web component. This looks good otherwise.